src/components/hosts/VMRebootConfigurationInfo.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Button, ButtonVariant, Popover, Text, TextVariants } from '@patternfly/react-core';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { global_warning_color_100 as warningColor } from '@patternfly/react-tokens';
+import { Host, stringToJSON, Inventory } from '../../api';
+
+const Hint = () => (
+  <>
+    <Text component={TextVariants.p}>
+      Your libvirt virtual machines should be configured to restart automatically after a reboot.
+      You can check this by running:
+    </Text>
+    <Text component={TextVariants.p}>
+      <code>
+        virsh dumpxml {'<'}name{'>'} | grep on_reboot{' '}
+      </code>
+    </Text>
```

Comment:
It would be better to use `PrismCode` component for code snippets. See `DiscoveryTroubleshootingModal` component.

Reply:
Done. What about changing the default theme to something better fitting the rest of the page? https://github.com/PrismJS/prism-themes
But in a follow-up.